framework/origin_trials_client.py

```diff
@@ -28,6 +28,35 @@
 import settings
 
 
+class RequestTrial(TypedDict):
+  display_name: str
+  start_milestone: str
+  end_milestone: str
+  end_time: dict
+  description: str
+  documentation_url: str
+  feedback_url: str
+  intent_to_experiment_url: str
+  chromestatus_url: str
+  allow_third_party_origins: bool
+  type: str
+  origin_trial_feature_name: NotRequired[str]
+
+
+class InternalRegistrationConfig(TypedDict):
+  allow_public_suffix_subdomains: NotRequired[bool]
+  approval_type: NotRequired[str]
+  approval_buganizer_component_id: NotRequired[int]
+  approval_buganizer_custom_field_id: NotRequired[int]
+  approval_criteria_url: NotRequired[str]
+  approval_group_email: NotRequired[str]
+
+
+class CreateOriginTrialRequest(TypedDict):
+  trial: RequestTrial
+  internal_registration_config: InternalRegistrationConfig
```

Comment:
Note that I created these fields on the OT server side:
https://source.corp.google.com/piper///depot/google3/google/internal/chrome/origintrials/v1/trials_integration.proto;l=31

Reply:
Awesome, I'll add them in a separate PR. 🙂